src/database/queries/userExtension.js

```diff
@@ -298,4 +298,28 @@ module.exports = class MenteeExtensionQueries {
 			return error
 		}
 	}
+
+	static async getMenteesExtensions(userIds, attributes = []) {
+		try {
+			const queryOptions = { where: { user_id: { [Op.in]: userIds } }, raw: true }
+			if (attributes.length > 0) {
+				queryOptions.attributes = attributes
+			}
+			const mentees = await MenteeExtension.findAll(queryOptions)
+			const menteeMap = new Map(mentees.map((mentee) => [mentee.user_id, mentee]))
+			const validMentees = []
+			const invalidMentees = []
+			userIds.forEach((userId) => {
+				const mentee = menteeMap.get(userId)
+				if (mentee) {
+					validMentees.push(mentee)
+				} else {
+					invalidMentees.push(userId)
+				}
+			})
```

Comment:
We cannot have these here. Functions inside queriers should return DB response as it is.

Reply:
ok can i use it in create session as a function ? @nevil-mathew

Reply:
You can move this to the appropriate place, but this has to be moved from queries.